### ISSUE TITLE
Add Baubles Compat

### ISSUE THREAD

**Issue by Syampuuh** [TinyModularThings/Player-Weight#16]
While digging through a bunch of backpack mods to see what this mod interacts with, I found a few issues with two mods that it otherwise mostly works with.

With [Improved Backpacks](https://www.curseforge.com/minecraft/mc-mods/improvedbackpacks), the mod doesn't see the backpacks' contents at all when it's equipped in the Bauble slot rather than the chest armor slot.
Demonstration:
With [Wearable Backpacks](https://www.curseforge.com/minecraft/mc-mods/wearable-backpacks) the mod has trouble recognizing when the backpack is dropped/equipped and updating the player's weight accordingly, which isn't too surprising considering the weird non-inventory ways it's interacted with.
Demonstrations of both: https://streamable.com/usbfr

If either of these seem like they're more of an issue with those mods compat than anything you can fix on your side, please let me know so I can let them know.

Also: As just kind of a general suggestion, it would be neat if there were a way to set a multiplier for item weights in a backpack, such as items being half as heavy in a bag to encourage their use.

**Comment by Speiger** [comment]
yeah i get what you mean. But there isnt a baubles plugin right out of the gate. though it would be a simple addition.

**Comment by Syampuuh** [comment]
Ah, so it's just an issues with Baubles slots in general huh. I didn't think to check the first time, but yeah it does seem that any item in a Baubles slot doesn't get its weight counted. It'd be greatly appreciated if you could add support for it some time.

By the way, rather than making another issue I figure I may as well mention here that I requested support for your mod with the Tool Belt mod, and the developer suggested that it may make more sense for you to use the Forge IItemHandler: https://github.com/gigaherz/ToolBelt/issues/59

**Comment by Speiger** [comment]
Eh I am doing that actually already. And belts should work as long they are not in bauble slots.
The issue here is just that the logic isnt checking the "Bauble" inventory.
So as soon a bauble compat comes in (that makes the bauble inventory visible to playerweight) that should be automatically supported.

Gigaherz is right there he shouldnt do anything.

Ok to put my standpoint here:
I have right now like 1/10th of the time that I had 1 month ago. And all time I have is right now on projects that need it more. 
Chunk Pregenerator and Unique Enchantments are some of those. (Each of them just get a day (even so they would need more time) the rest is on other stuff that is way more important)
So It might take a bit before i have time to work on this project even if the change is small.

**Comment by Syampuuh** [comment]
Sounds good, no rush. Appreciate it.
